hooks/command

```diff
@@ -1,575 +1,23 @@
 #!/bin/bash
 set -euo pipefail
 
-# retry <number-of-retries> <command>
-function retry {
-  local retries=$1; shift
-  local attempts=1
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
-  until "$@"; do
-    retry_exit_status=$?
-    echo "Exited with $retry_exit_status"
-    if (( retries == "0" )); then
-      return $retry_exit_status
-    elif (( attempts == retries )); then
-      echo "Failed $attempts retries"
-      return $retry_exit_status
-    else
-      echo "Retrying $((retries - attempts)) more times..."
-      attempts=$((attempts + 1))
-      sleep $(((attempts - 2) * 2))
-    fi
-  done
-}
+# shellcheck source=lib/shared.bash
+. "$DIR/../lib/shared.bash"
 
-# Reads a list from plugin config into a global result array
-# Returns success if values were read
-plugin_read_list_into_result() {
-  result=()
+commands=()
 
-  for prefix in "$@" ; do
-    local i=0
-    local parameter="${prefix}_${i}"
+[[ -n "$(plugin_read_list LOAD)" ]] && commands+=("LOAD")
+[[ -n "$(plugin_read_list IMAGE)" ]] && commands+=("IMAGE")
```

Comment:
if you follow my suggestion (about the loading being conditional but not the run portion) this array and subsequent logic would not be necessary